roles/os_must_gather/tasks/main.yml

```diff
@@ -69,10 +69,25 @@
           SOS_DECOMPRESS=$SOS_DECOMPRESS
           gather &> {{ cifmw_os_must_gather_output_dir }}/logs/os_must_gather.log
 
+    # directory name will be generated starting from cifmw_os_must_gather_image
+    # variable e.g.:
+    # EXAMPLE 1
+    # original value: "quay.io/openstack-k8s-operators/openstack-must-gather:latest"
+    # pattern value: "quay-io-openstack-k8s-operators-openstack-must-gather*"
+    # EXAMPLE 2
+    # original value: "foo.bar.example.com/repofoo/openstack-must-gather-rhel9:1.0.0"
+    # patterns value: "foo-bar-example-com-repofoo-openstack-must-gather-rhel9*"
```

Comment:
I'm slightly worried by the potential "path too long" issue if for any reason the leading directory (that one, here) has a extensively long name. A follow-up truncating name might be something to consider?

Reply:
@cjeanner we are not creating that folder, in fact we move away that long name to a "sane" name. We're only searching for it in that task. I guess that the folder is generated by must-gather itself.